.github/workflows/tag-on-merge.yml

```
@@ -0,0 +1,25 @@
+name: Tag on merge to master
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Compute tag name
+      id: compute-tag
+      run: |
+        echo Computing next tag number
+        LASTPATCH=$(git describe --tags | cut -d- -f1 | cut -d. -f3)
+        PATCH=$(($LASTPATCH+1))
+        echo "::set-output name=tagname::2.1.${PATCH})"
+
+    - name: Create Tag
```

```suggestion
    - name: Create tag
```

We don't use a lowercase `tag` in the other `name:` in this file, so should probably be consistent across both names. 😉